unlock-app/src/components/interface/checkout/Stepper.tsx

```diff
@@ -117,6 +122,10 @@ export const Stepper = ({
             <StepButton
               key={idx}
               onClick={() => {
+                if (item.to === 'CONNECT') {
```

Comment:
I am not sure this is the right place to do that. Can you do it in the state machine itself?